devenv/docker/blocks/slow_proxy/main.go

```diff
@@ -16,13 +15,21 @@ func main() {
 		origin = "http://localhost:9090/"
```

Comment:
Should this read the `ORIGIN_SERVER` env var?

Reply:
It does 2 lines above?

Reply:
Damn GitHub diff hiding it
![FistShake](https://media.giphy.com/media/X1nICIXRE8JVu/giphy.gif)

Reply:
:smile: